### ISSUE TITLE
Scrolling during custom block creation creates text misalignment 

### ISSUE THREAD

**Issue by NitroCipher** [LLK/scratch-gui#3105]
Two finger touchpad scrolling causes the text input field to become misaligned with the block being created. I have yet to test this with a wheeled mouse. I will report back later

![screenshot 2018-09-10 at 09 09 16](https://user-images.githubusercontent.com/11393734/45302628-69c19f80-b4d9-11e8-932c-8361e2035009.png)

My browser / operating system: ChromeOS 10575.58.0, Chrome 67.0.3396.99, Flash 30.0 (release 0)

**Comment by MasterOfTheTiger** [comment]
Are you using natural scrolling or not?

**Comment by chexbox** [comment]
Doesn't happen with wheel on chrome.

**Comment by kyleplo** [comment]
Can't repro on Windows Chrome.

**Comment by MasterOfTheTiger** [comment]
Cannot reproduce on Firefox or Chrome on Fedora GNU/Linux.

**Comment by NitroCipher** [comment]
I can't even reproduce it myself. It seems to be a ChromeOS isolated issue

**Comment by MasterOfTheTiger** [comment]
Very very odd. The Chrome on ChromeOS should behave the same way as on other platforms right?

**Comment by Kenny2github** [comment]
Can't repro on Windows 10 Chrome 68

**Comment by chexbox** [comment]
Can't reproduce with scrollbars on chromeOS. Dialog just disappears.

**Comment by NitroCipher** [comment]
It is really strange. It only happens with the two finger scrolling gesture on the mousepad. (chromeOS exclusive bug)

**Comment by chexbox** [comment]
Successfully repo.
![screenshot 2018-09-13 at 10 39 14](https://user-images.githubusercontent.com/31783961/45499119-531d8180-b741-11e8-8104-78f61ac25dff.png)
![screenshot 2018-09-13 at 10 40 15](https://user-images.githubusercontent.com/31783961/45499156-6e888c80-b741-11e8-9a6c-adbe05335d91.png)
The text-box scrolls but the rest of the dialog does not.

**Comment by MasterOfTheTiger** [comment]
@chexbox How small in pixels does the screen have to be before the bug occurs?

**Comment by NitroCipher** [comment]
@MasterOfTheTiger Yeah, just reproduced it myself on an iMac, and then your comment appeared. Smaller than 1024x640 is the glitches sweet spot. Updating the issue in a sec

**Comment by chexbox** [comment]
Successful repro on raspberry pi Chrome, and Firefox.
